src/content/docs/dropins/order/functions.mdx

```diff
@@ -8,3 +8,542 @@ tableOfContents:
+    ['queryType', 'QueryType', 'No', 'The type of query to perform. Use orderData as the value'],
+    ['returnsPageSize', 'number', 'No', 'The number of returns to return at a time.'],
+  ]}
+/>
+
+### Returns
+
+Returns a promise that resolves to a `TransformedData` object.
+
+### Usage
+
+type QueryType = 'orderData';
+
+```ts
+getOrderDetailsById(
+  orderId?: 12345;
+  queryType: QueryType;
+);
+```
+
+## getStoreConfig
+
+The `getStoreConfig` function returns information about the storefront configuration. It is a wrapper for the [`storeConfig`](https://developer.adobe.com/commerce/webapi/graphql/schema/store/queries/store-config/) query.
+
+```ts
+export const getStoreConfig = async (): Promise<StoreConfigModel | null>
+```
+
+### Returns
+
+Returns a promise that resolves to a `StoreConfigModel` object or null.
+
+```ts
+type StoreConfigModel = {
+  order_cancellation_enabled: boolean;
+  order_cancellation_reasons: {
+    description: string;
+  };
+};
+```
+
+### Usage
+
+The following example demonstrates how to retrieve the store configuration:
+
+```ts
+getStoreConfig();
+```
+
+## guestOrderByToken
+
+The `guestOrderByToken` function retrieves a guest order using a token generated by Adobe Commerce. It is a wrapper for the `guestOrderByToken` query.
+
+```ts
+export const guestOrderByToken = async (
+  token?: string,
+  returnRef?: string
+): Promise<OrderDataModel | null>
+```
+
+<OptionsTable
+  compact
+  options={[
+    ['Parameter', 'Type', 'Req?', 'Description'],
+    ['token', 'string', 'No', 'A token for the order assigned by Adobe Commerce.'],
+    ['returnRef', 'string', 'No', 'The reference to return.'],
+  ]} 
+/>
+
+### Returns
+
+Returns a promise that resolves to an `OrderDataModel` object or null.
+
+<CodeInclude code={orderDataModel} lang="ts" frame="none" />
+
+### Usage
+
+```ts
+guestOrderByToken(token:'abcde12345');
+```
+
+## reorderItems
+
+The `reorderItems` function allows a logged-in customer to add all the products from a previous order into their cart. It is a wrapper for the [`reorderItems`](https://developer.adobe.com/commerce/webapi/graphql/schema/orders/mutations/reorder-items/) mutation.
+
+```ts
+export const reorderItems = async (
+  orderNumber: string
+): Promise<ReorderItemsProps>
+```
+
+<OptionsTable
+  compact
+  options={[
+    ['Parameter', 'Type', 'Req?', 'Description'],
+    ['orderNumber', 'string', 'Yes', 'The order number to reorder.'],
+  ]}
+/>
+
+### Returns
+
+Returns a promise that resolves to a `ReorderItemsProps` object.
+
+```ts
+export interface ReorderItemsResponse {
+  data: {
+    reorderItems: {
+      cart: {
+        itemsV2: {
+          items: { uid: string }[];
+        };
+      };
+      userInputErrors: UserInputErrorProps[];
+    };
+  };
+
+  errors?: {
+    message: string;
+  }[];
+}
+```
+
+### Usage
+
+The following example demonstrates how to reorder items from a previous order:
+
+```ts
+reorderItems(orderNumber: '12345');
```

Comment:
Could you change type orderId?: string;, please. Thank you.
If you wanted to use the example order 000004355